contracts/ethereum/contracts/LootboxStaking.sol

```diff
@@ -99,6 +99,8 @@ contract LootboxStaking is ILootboxStaking, IOperatorOwned, IEmergencyMode {
     function init(
         address _operator,
         address _emergencyCouncil,
+        uint256 _minLockupTime,
```

Comment:
Need a separate function to initialise these variables and need to bump up the version